### ISSUE TITLE
Multiple 'ghost' instances of a mod. 

### ISSUE THREAD

**Issue by Nyraen** [Nexus-Mods/Vortex#12031]
Tried to uninstall, unresponsive, seems to bug the game nontheless (texture is totaly bugged). Tried to resintall to 'capture' the leftover, failed (might be a version problem, not the right version to capture that). No idea what to try next, can't find anything in vortex files to clean (nothing clear enough to avoid doing worse) 
![ghost](https://user-images.githubusercontent.com/45366647/172660859-d16e6ede-d8cd-442e-a74d-4c350e84a3d8.png)

EDIT : tried to clear the cache (old school mode : deleted the folder)
Deleted all previous download of mod that as a problem, reinstalled fresh 
PS : yes I know I could have done that 5min earlier, let that be an archive for people in need

**Comment by TanninOne** [comment]
@Nyraen You didn't say what happened after you deleted the folders, are you saying the mods are now gone?
Otherwise: Please right-click the mod, select "Open in File Manager" and see whether they are actually the same mod. I'm kind of suspecting they are different mods that for some reason have the same meta data associated with them but unless you know how to reproduce this there is nothing we can investigate.

**Comment by Nyraen** [comment]
First thing : deleting the cash removed the 'ghost' instances. 
As for the reproducing part I'm not sure. First I changed my game drive (moved from D to C), and reinstalled the mods there (I don't think I touched the mod for the suppression part, I expected them to be dealt with by the uninstall, but I can't say for sure if I had to uninstall or disable to pass them on the new install). I also had trouble with a bug in texture (probably coming from somewhere else as this whole operation didn't solve the problem), but this mean I tried to remove some mods, seeing the ghost, I tried to reinstall uninstall to capture the old file but that didn't work. Another Subtlety was that I recently updated the mods that weren't up to date (and I think the concerned mod was among them). Sorry to not be able to be more precise.
